packages/shared/src/UI/components/AddCollectibles/index.tsx

```diff
@@ -132,10 +144,20 @@ export const AddCollectibles = memo(function AddCollectibles(props: AddCollectib
     const hub = useWeb3Hub(pluginID, { chainId })
     const connection = useWeb3Connection(pluginID, { chainId })
 
+    const { value: addressType, loading: loadingAddressType } = useAddressType(
+        pluginID,
+        Others.isValidAddress?.(address) ? address : '',
+        {
+            chainId,
+        },
+    )
+
     const validationMsgForAddress = useMemo(() => {
-        if (!Others.isValidAddress?.(address ?? '')) return t.collectible_contract_invalid()
+        if (!address) return ''
+        if (!Others.isValidAddress?.(address ?? '') || (addressType !== AddressType.Contract && !loadingAddressType))
```

Comment:
```suggestion
        if (!Others.isValidAddress?.(address) || (addressType !== AddressType.Contract && !loadingAddressType))
```

Reply:
No, if the textfield is empty, it shouldn't display error message.